panels/config/config-entries/ha-config-entries.html

```diff
@@ -87,11 +88,15 @@ <h1>Add integration</h1>
       return {
         hass: Object,
 
-        flowId: {
+        _flowId: {
           type: String,
           value: null,
         },
-        userCreatedFlow: Boolean,
+        /*
+         * The step of the current selected flow, if available.
+         */
+        _flowStep: Object,
+        _userCreatedFlow: Boolean,
```

Comment:
This is not used in bindings. Maybe make it a non-polymer property?

Reply:
Fixed